azure-pipelines.yml

```diff
@@ -941,15 +941,6 @@ jobs:
       arguments: -c ${{ parameters.BuildConfiguration }} -r osx-x64 --no-restore --self-contained false --verbosity detailed
       zipAfterPublish: false
       modifyOutputPath: false
-  - task: DotNetCoreCLI@2
```

Comment:
Am I correct in understanding this (and the copying/packaging/publishing steps) are removed here because they're being done in a different job running in a macos pool?

Reply:
Yes that is correct.